arviz/tests/base_tests/test_data.py

```diff
@@ -77,6 +79,18 @@ def no_remote_data(monkeypatch, tmpdir):
     )
 
 
+@pytest.fixture
```

Comment:
what was the issue with module scope? I think all the other fixtures use it.

Reply:
The tests were failing for some cases in tests which used `inplace` like `isel`, as `inplace=True`is changing the data and datarandom was being generated only once if the scope is module. I changed the scope to function (default) so that data is generated everytime.